protocol/messenger_mailserver_cycle.go

```diff
@@ -434,14 +434,7 @@ func (m *Messenger) connectToMailserver(ms mailservers.Mailserver) error {
 
 			// Query mailserver
 			if m.config.codeControlFlags.AutoRequestHistoricMessages {
-				go func() {
-					_, err := m.performMailserverRequest(&ms, func(_ mailservers.Mailserver) (*MessengerResponse, error) {
-						return m.RequestAllHistoricMessages(false, false)
-					})
-					if err != nil {
-						m.logger.Error("could not perform mailserver request", zap.Error(err))
-					}
-				}()
+				m.asyncRequestAllHistoricMessages()
```

Comment:
`asyncRequestAllHistoricMessages` doesn't use `performMailserverRequest`.
Not sure if we want it, but noting the difference.

Reply:
Sorry, this one i don't quite understand - `asyncRequestAllHistoricMessages` does invoke `performMailserverRequest` here https://github.com/status-im/status-go/blob/e1e04e6f92be0515b6ada4a5143e4728e02991be/protocol/messenger_mailserver.go#L362

Reply:
oh, I got confused
So we were calling `performMailserverRequest` -> `RequestAllHistoricMessages` -> `performMailserverRequest` 😵‍💫

I guess then it's ok now